x/events/events.go

```diff
@@ -42,6 +43,7 @@ const (
 	WebhookDelivered         semconv.Event = "WebhookDelivered"
 	WebhookSucceeded         semconv.Event = "WebhookSucceeded"
 	WebhookFailed            semconv.Event = "WebhookFailed"
+	ClaimsMappingFailed      semconv.Event = "ClaimsMappingFailed"
```

Comment:
Any alternatives to the naming?

Reply:
Hmm, prefix with `OIDC` for context? I.e. `OIDCClaimsMappingFailed` ?